assets/js/modules/analytics/components/settings/GA4DashboardViewToggle.js

```diff
@@ -59,24 +64,29 @@ export default function GA4DashboardViewToggle() {
 		);
 	}, [ dashboardView, setDashboardView, viewContext ] );
 
+	const displayDashboardView = isUAAvailable
+		? dashboardView
+		: DASHBOARD_VIEW_GA4;
+
 	return (
 		<div>
 			<Switch
 				label={ __(
 					'Display metrics from Google Analytics 4 on the dashboard',
 					'google-site-kit'
 				) }
-				checked={ dashboardView === 'google-analytics-4' }
+				checked={ displayDashboardView === 'google-analytics-4' }
```

Comment:
Probably a simple oversight from before but this should also use the constant.

```suggestion
				checked={ displayDashboardView === DASHBOARD_VIEW_GA4 }
```

Reply:
Good spot, thanks. Have updated :+1: